crypto/BRCryptoWalletManagerClient.c

```diff
@@ -652,38 +652,6 @@ cwmTransactionEventAsBTC (BRWalletManagerClientContext context,
                 cryptoUnitGive (unit);
            }
 
-            // ... update state to reflect included if the timestamp and block height are already set
-            if (0 != btcTransaction->timestamp && TX_UNCONFIRMED != btcTransaction->blockHeight) {
-
```

Comment:
Maybe a comment in the 'empty code' to the effect: "We won't announce a state change because ..."